src/github/orbit-action.test.js

```diff
@@ -208,12 +208,14 @@ test("should create content for existing members", async () => {
     expect(getByText(orbitDetailsElement, "Adding the content…"));
   });
   expect(global.fetch).toHaveBeenCalledWith(
-    expect.stringContaining("/my-workspace/members/phacks/posts"),
+    expect.stringContaining("/my-workspace/members/phacks/activities"),
     expect.objectContaining({
       body: JSON.stringify({
-        url:
-          "https://github.com/orbit-love/orbit-model/issues/10#issuecomment-590037251",
-        published_at: "2020-02-23T07:55:28Z",
+        activity_type: "content",
+        url: "https://github.com/orbit-love/orbit-model/issues/10#issuecomment-590037251",
+        title: "Commented on GitHub",
```

Comment:
I believe `title` and `description` are not supported for ContentActivities—the “fancy” display in the UI is generated by a link thumbnailer in the backend